crates/parcel_plugin_rpc/src/nodejs/rpc_conns_nodejs.rs

```diff
@@ -0,0 +1,38 @@
+use parking_lot::Mutex;
+
+use super::RpcConnectionNodejs;
+
+use crate::RpcConnection;
+
+/// Connection to multiple Nodejs Workers
+/// Implements round robin messaging
+pub struct RpcConnectionsNodejs {
+  current_index: Mutex<usize>, // TODO use AtomicUsize
+  conns: Vec<RpcConnectionNodejs>,
+}
+
+impl RpcConnectionsNodejs {
+  pub fn new(conns: Vec<RpcConnectionNodejs>) -> Self {
+    Self {
+      current_index: Default::default(),
+      conns,
+    }
+  }
+
+  fn next_index(&self) -> usize {
+    let mut current_index = self.current_index.lock();
+    if *current_index >= self.conns.len() - 1 {
+      *current_index = 0;
+    } else {
+      *current_index = *current_index + 1;
+    }
+    current_index.clone()
+  }
+}
```

Comment:
I tried to use an `AtomicUsize` for this initially but I wasted too much time trying to figure out the `fetch_update`. 

I assume we will change this in future to schedule work on workers based on if they are busy so for now round robin with a Mutex<usize> is fine